maint/code-cleanup.sh

```diff
@@ -118,6 +146,7 @@ for arg in $@; do
 	ignore=1
     else
 	got_file=1
+        filelist="$filelist $arg"
```

Comment:
This is a little deceiving. It's not legal to pass more than one file in here.

Reply:
This is what I was wondering -- why not allow passing multiple files?

Reply:
I don't have a problem with it, but it'll require a little more work.

Reply:
I tested locally with multiple files, seems working. Which part do you think need more work?

Reply:
Sorry. I'd missed that you'd added the `for` loop. Then this should be fine. 👍